### ISSUE TITLE
Unsupported Kernel type for "aws:import_image" step

### ISSUE THREAD

**Issue by cprovencher** [rjocoleman/oracle-linux-ami-builder#2]
@rjocoleman  Have you seen this issue before?
I've been getting an error in the "aws:import_image" step:
ClientError: Unsupported kernel version 4.1.12-94.3.9.el7uek.x86_64
I also have not chosen that kernel during boot.
I am using the iso for oracle linux 7.4

**Comment by rjocoleman** [comment]
Good question! 

I haven't used this in a wee-while and reviewing the code it looks like the whole thing was predicated on the [first grub option not being UEK](https://github.com/rjocoleman/oracle-linux-ami-builder/blob/6590dfd41dfb4fed6dba433e9eaf0d69620d8e3f/scripts/ec2.sh#L8-L10). 

I'll take a shot at updating this code base to 7.4 real quick and see what happens.

**Comment by cprovencher** [comment]
@rjocoleman  thanks! If you want to save some time just use my fork: https://github.com/cprovencher/oracle-linux-ami-builder
It's already set up for 7.4.
I didn't think those grub commands were the culprit because I manually picked another kernel and that ec2.sh script is only executed after the first boot. I made sure the right kernel would be picked on first boot as well https://github.com/cprovencher/oracle-linux-ami-builder/blob/master/http/ks.cfg#L77
